snuba/web/converters.py

```diff
@@ -0,0 +1,12 @@
+from werkzeug.routing import BaseConverter
+
+from snuba.datasets.dataset import Dataset
+from snuba.datasets.factory import get_dataset
+
+
+class DatasetConverter(BaseConverter):
+    def to_python(self, value: str):
+        return get_dataset(value)
+
+    def to_url(self, value: Dataset):
+        raise NotImplementedError
```

Comment:
We don't have a way to easily map dataset instances back to their names and we don't really use any sort of URL building right now. Probably should fix this later but not a critical concern, IMO.

Reply:
Maybe add a comment mentioning that this is not implemented because we do not need it now, and if someone needs it, it just have to be implemented.

Reply:
Added here: 1fc2204f5fc7dd5f88917649a146197e4e53a37a